src/features/FeatureTour/FeatureTour.tsx

```diff
@@ -105,7 +105,7 @@ export const FeatureTour = ({
         styles={{
           options: {
             arrowColor,
-            zIndex: 2000, // need this for it to be on top of the navbar
+            zIndex: 1100, // need this for it to be on top of the navbar
```

Comment:
@QiluXie The hotfix PR tried to monitor the state as to if a modal is open or not to determine whether we need to show the feature tour. Bu that is not required.

The way we should handle this is by ensuring modals have a higher zIndex (which means they overlay the feature tour).

At the same time, we need to ensure feature tour has a zIndex which is more than the nav bar and the dashboard/workspace components.

1100 is a sweet spot between the 2 values which will allow feature tour to show up on dashboard/workspace "above" the components and will hide feature tour when a modal is opened (since modals will have a higher zIndex)

Reply:
![Screenshot 2023-04-28 at 6 39 22 PM](https://user-images.githubusercontent.com/5507822/235127015-b270fcb4-ac5f-4dff-be2d-40b6b147a862.png)

![Screenshot 2023-04-28 at 6 40 29 PM](https://user-images.githubusercontent.com/5507822/235127040-5196cda4-9b32-4b93-82a8-bbd292797c7a.png)

Review Request modal has a zIndex of 1400
s a z

Reply:
![Screenshot 2023-04-28 at 6 42 35 PM](https://user-images.githubusercontent.com/5507822/235127432-4107b69c-c1b2-4367-94e8-d62ab7968d1e.png)

Create new folder has 1300

Reply:
I tried using 1200, but it didn't work as there are other components with 1200. Hence 1100 was the option I went with